civiform/civiform
# Spliting of Checkbox question options into its own column for CSV export

### Description
All changes needed to split Checkbox question options into its own column for CSV export.

## Release notes

Spliting of Checkbox question options into its own column for CSV export.

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)



### Instructions for manual testing

1. Pull this branch to your local machine
2. Run bin/dev-run. Create Checkbox question and program using it.
3. Login as applicant or guest and sub mit applications to the program.
4. Login as Program admin and download the applications as CSV.
5. Each option in the Checkbox question should appear in its own column.

### Issue(s) this completes

Fixes #5020
